docs/concepts/trading-on-vega/positions-margin.md

```diff
@@ -1,32 +1,26 @@
-
-`[margin account balance] = [initial margin requirement] + [unrealised profit] OR - [unrealised losses]`
-
-## Automated market mechanisms 
+## Automated market processes
 As markets and collateral are not managed through human intervention, markets must have certain automated processes that allow them to function well and assure that the collateral required to manage positions is available when it's needed.
 
-There are a few mechanisms that work differently to how they would on a centralised exchange, in order to keep the markets solvent. They include:
-- [**Margin**](#margin): When a party opens a position: the *initial margin* requirement is calculated automatically depending on the market's risk model. If the market moves against the party, and the margin towards the *maintenance level*, Vega will *search* for more collateral in the general account, to avoid liquidating the position. Margin can also be *released* if the position is in sufficient profit. Other positions in markets with the same settlement asset may also interact with the same general account. Therefore, Vega is a cross-margin based system by default.
+There are a few mechanisms that work differently to how they would on a centralised exchange, in order to keep the markets solvent. 
+
+They include:
+- [**Cross margining**](#cross-margining): When a participant places an order using cross margin mode, the *initial margin* requirement is calculated automatically depending on the market's risk model. If the market moves against the participant, and the margin towards the *maintenance level*, Vega will *search* for more collateral in the general account, to avoid liquidating the position. Margin can also be *released* if the position is in sufficient profit. Other positions in markets with the same settlement asset may also interact with the same general account. 
+- [**Margin isolated per position**](#isolated-margin): When a participant places an order using isolated margin mode, the expected margin required for the life of the order, if it's filled, is set aside. The network continually tracks the requirements for open orders and positions to ensure there is enough margin to keep them open.
 - [**Mark to market**](#mark-to-market): Mark to market on Vega happens much more frequently than typical exchanges. Every time a trade happens and moves the last traded price, positions are marked to market. Marking to market is used to move assets into your margin account (from someone else's) if you are in profit, or out of your margin account if not.
```

Comment:
the MTM frequency is controlled by a network parameters: network.markPriceUpdateMaximumFrequency